_includes/main_menu.html

```diff
@@ -1,144 +1,165 @@
+            <span class=dropdown-title>Models</span>
+            <p>Discover, publish, and reuse pre-trained models</p>
+          </a>
+          <a class="nav-dropdown-item" href="{{ site.external_urls.discuss }}" target="_blank">
+            <span class=dropdown-title>Forums</span>
+            <p>A place to discuss PyTorch code, issues, install, research</p>
+          </a>
+          <a class="nav-dropdown-item" href="{{ site.baseurl }}/resources">
+            <span class=dropdown-title>Developer Resources</span>
+            <p>Find resources and get questions answered</p>
+          </a>
+          <a class="nav-dropdown-item" href="{{ site.baseurl}}/ecosystem/contributor-awards-2023">
+            <span class="dropdown-title">Contributor Awards - 2023</span>
+            <p>Award winners announced at this year's PyTorch Conference</p>
+          </a>
         </div>
       </div>
     </li>
 
     <li class="main-menu-item">
       <div id="dropdownMenuButton" data-toggle="resources-dropdown" class="resources-dropdown">
-        <a class="resource-option {% if (current[1] == {{ site.baseurl }} or current[1] == 'foundation' or current[1] == 'join' or current[1] == 'edge' or current[1] == 'executorch-overview') %}with-down-white-arrow{% else %}with-down-arrow{% endif %}">
+        <a class="with-down-arrow">
           Edge 
         </a>
         <div class="resources-dropdown-menu">
           <a class="nav-dropdown-item" href="{{ site.baseurl}}/edge">
             <span class="dropdown-title">About PyTorch Edge</span>
+            <p>Build innovative and privacy-aware AI experiences for edge devices</p>
           </a>
           <a class="nav-dropdown-item" href="{{ site.baseurl}}/executorch-overview">
             <span class="dropdown-title">ExecuTorch</span>
+            <p>End-to-end solution for enabling on-device inference capabilities across mobile and edge devices</p>
           </a>
         </div>
       </div>
     </li>
 
-    <li class="main-menu-item {% if current[1] == 'blog' %}active{% endif %}">
-      <a href="{{ site.baseurl }}/blog">Blog</a>
-    </li>
-
-    <li class="main-menu-item">
-      <a href="{{ site.external_urls.tutorials }}">Tutorials</a>
-    </li>
-
     <li class="main-menu-item">
       <div id="docsDropdownButton" data-toggle="resources-dropdown" class="resources-dropdown">
-        <a class="resource-option {% if (current[1] == {{ site.baseurl }} or current[1] == 'foundation' or current[1] == 'join' or current[1] == 'edge' or current[1] == 'executorch-overview') %}with-down-white-arrow{% else %}with-down-arrow{% endif %}">
+        <a class="with-down-arrow">
           Docs
         </a>
         <div class="resources-dropdown-menu">
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/docs">
+          <a class="nav-dropdown-item" href="https://pytorch.org/docs">
             <span class="dropdown-title docs-title">PyTorch</span>
             <p></p>
```

Comment:
Should we add text here for consistency with other dropdowns?
```suggestion
            <p>Explore the documentation for comprehensive guidance on how to use PyTorch.</p>
```